Packs/CommonPlaybooks/Playbooks/playbook-Ticket_Management_-_Generic.yml

```diff
@@ -0,0 +1,703 @@
+          root: inputs.addCommentPerEndpoint
+      description:
+        complex:
+          root: inputs.description
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 66b87a8c-c5a1-489c-8cef-ea4c198faa4e
+    type: title
+    task:
+      id: 66b87a8c-c5a1-489c-8cef-ea4c198faa4e
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 870,
+          "y": 950
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 159abeff-fe4e-42fc-8be0-67f4ba0f245b
+    type: regular
+    task:
+      id: 159abeff-fe4e-42fc-8be0-67f4ba0f245b
+      version: -1
+      name: Set Zendesk Ticket ID
+      description: commands.local.cmd.set.parent.incident.context
+      script: Builtin|||setParentIncidentContext
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      key:
+        simple: ZendeskTicketID
+      value:
+        complex:
+          root: alert
+          accessor: id
+          transformers:
+          - operator: If-Then-Else
+            args:
+              condition:
+                value:
+                  simple: lhs==rhs
+              conditionB: {}
+              conditionInBetween: {}
+              else:
+                value:
+                  simple: Zendesk.Ticket.id
+                iscontext: true
+              equals: {}
+              lhs:
+                value:
+                  simple: Zendesk.Ticket.id
+                iscontext: true
+              lhsB: {}
+              options: {}
+              optionsB: {}
+              rhs: {}
+              rhsB: {}
+              then: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: a27c8fa0-c258-41db-84e9-8b93a26c1623
+    type: regular
+    task:
+      id: a27c8fa0-c258-41db-84e9-8b93a26c1623
+      version: -1
+      name: Set ServiceNow Ticket ID
+      description: commands.local.cmd.set.parent.incident.context
```

Comment:
Maybe add a more informational description explaining the if-then-else check.

Reply:
Done.